disnake/types/components.py

```diff
@@ -72,7 +74,7 @@
 
 
 class _BaseComponent(TypedDict):
-    # type: ComponentType  # FIXME: current version of pyright only supports PEP 705 experimentally, this can be re-enabled in 1.1.353+
+    type: ReadOnly[ComponentType]
```

Comment:
Out of scope. Please submit a separate change request.

Reply:
This isn't worth a separate PR, as it doesn't really fix anything. I can just remove it from this one, if you'd like.